api/api.go

```diff
@@ -146,6 +136,7 @@ func handleDownload(handler func(r *http.Request) (io.ReadCloser, map[string]str
 			if err != nil {
 				log.Error(ctx, "handleDownload: Error while copying from reader", err, log.Data{"request:": request})
 				http.Error(w, err.Error(), http.StatusInternalServerError)
+				return
```

Comment:
Not sure about this - there's only a return statement in this branch. Presuming the linter complained?